.github/workflows/3d-city-skyline.yml

```
@@ -0,0 +1,489 @@
+name: 3D City Skyline Performance
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)